client/app/components/_pickupList/pickupListItem/pickupListItem.controller.js

```diff
@@ -1,12 +1,15 @@
 class PickupListItemController {
   constructor(PickupDate, User) {
     "ngInject";
-    this.PickupDate = PickupDate;
-    this.User = User;
-    this.showStoreDetail = this.showDetail === "store";
-
-    this.collectors = [];
+    Object.assign(this, {
+      PickupDate,
+      User,
+      collectors: []
+    });
+  }
 
+  $onInit() {
```

Comment:
@D0nPiano just as a short example for the changes in angular 1.6: the setCollectors/setStoreInfo functions need to be called from $onInit instead of the constructor, because they need the data bindings.

(Another way could be: put them in a $onChanges hook, then they will get updated when the binding changes. It's generally the more flexible way :) )